client/js/shout.js

```diff
@@ -15,6 +15,7 @@ $(function() {
 		"/nick",
 		"/notice",
 		"/op",
+		"/otr",
```

Comment:
Let's consider subcmd completion in a second time, maybe open a ticket for that ?